src/test/java/com/cegeka/academy/service/InvitationServiceTest.java

```diff
@@ -140,5 +166,50 @@ public void assertThatRejectInvitationIsWorking() {
         assertThat(list.get(0).getEvent()).isEqualTo(invitation.getEvent());
     }
 
+    @Test
+    public void assertThatSendGroupInvitationToPrivateEventIsWorking() throws NotFoundException {
+
+        Long idGroup = groupRepository.findAll().get(0).getId();
+        invitationService.sendGroupInvitationsToPrivateEvents(idGroup, invitationSendToGroup);
+        List<Invitation> list = invitationRepository.findAll();
+        Optional<User> findUser = userRepository.findById(list.get(list.size() - 1).getUser().getId());
```

Comment:
Si aici cred ca trebui facuta o cautare.

Reply:
Optional<User> findUser = userRepository.findById(list.get(list.size() - 1).getUser().getId());
 Aici eu am facut deja o cautare dupa id user al ultimului element de tip invitation din lista. Nu imi dau seama ce cautare mai trebuie sa fac.
 Am inlocuit in modificarea de mai jos list.size() - 1 cu list.lastIndexOf(invitationSendToGroup). E mai bine asa ?

Reply:
Da, e bine. Buna idee!